tests/src/test/scala/system/basic/WskBasicPythonTests.scala

```diff
@@ -119,4 +122,37 @@ class CLIPythonTests
                     activation.logs.get.mkString("\n") should { not include ("pythonaction.py") and not include ("flask") }
             }
     }
+
+    behavior of "Python runtime"
+
+    it should "invoke a zipped Python action with virtualenv package" in withAssetCleaner(wskprops) {
```

Comment:
works with python3 using a zipped virtualenv that is built with python3
added the appropriate python3 test zip
will update documentation